src/installs/visualStudio.ts

```diff
@@ -170,6 +172,12 @@ export function getHostTargetArchString(hostArch: string, targetArch?: string, a
         targetArch = hostArch;
     }
 
+    const parsedTargetArch = targetArch.split(",");
+    const version = parsedTargetArch.filter(a => a.includes("version=")).map(a => a.split("=")[1]).join(" ").trimEnd();
```

Comment:
There is code in `preset.ts` that parses toolsets (`parseToolset`).  Perhaps that should be made an `export` and refactored to someplace common.

Reply:
Good idea, I'll do that before changing. Thanks for that catch.

Reply:
Actually, update, I'll leave any possible refactor for later because this isn't for parsing the toolset, it's for parsing the `architecture` presets field, not toolset.

Reply:
My bad.  It looks like that was introduced in 3.27 (after I stopped working on this).  In that case, you'd also want to update `getArchitecture` in `presets.ts` and make that the source of truth.

Reply:
Good idea, I'll make an issue to track that update.